### DISCUSSION TITLE
does yimmenu has a report protection?

### DISCUSSION THREAD

**Discussion by fakej1nx1e** [YimMenu/YimMenu#2484] [Q&A] [answered]
i just want to know

**Comment by Yimura** [comment, marked as answer]
Yes

**Reply by BlueGhost4** [reply]
Is that still the case for the new update or does the report protection still needs to be updated?

**Reply by Yimura** [reply]
> Is that still the case for the new update or does the report protection still needs to be updated?

Even if it doesn't work anymore why care if Rockstar doesn't ban you for it.

**Reply by BlueGhost4** [reply]
>Even if it doesn't work anymore why care if Rockstar doesn't ban you for it.
>
Some player do say that they reported me and i can't block it so i was just wondering. But you're right, as long as there is no ban there is no issue.
